tests/frontend/test_ops_decompositions.py

```diff
@@ -285,41 +284,20 @@ def test_merge(self, hbar, tol):
         S1 = random_symplectic(n)
         S2 = random_symplectic(n)
 
-        G1 = ops.GaussianTransform(S1, hbar=hbar)
```

Comment:
Do we have tests which test the hbar-dependence of the r parameter? (if I'm reading the comments correctly, it still depends on hbar)

Reply:
I added the r parameter to a few of the Gaussian tests.